modules/library/coverage/src/main/java/org/geotools/coverage/grid/io/footprint/MultiLevelROIProviderFactory.java

```diff
@@ -71,7 +90,7 @@ public static MultiLevelROIProvider createFootprintProvider(File referenceFile)
      * @throws Exception
      */
     public static MultiLevelROIProvider createFootprintProvider(
-            final File referenceFile, Geometry granuleBounds) {
+            final File referenceFile, Geometry granuleBounds, Hints hints) {
```

Comment:
The hints are not really used here, probably a leftover from a failed experiment?

Reply:
mmm nope. 
this implementation do not use hints, while the MultiLevelROIProviderMosaicFactory is actually using it

Reply:
But it's a static method, there is no notion of override...

Reply:
Ah! Right... nope... no reflection. I think that at the very beginning I was having issues with the MultiLevelROIProviderMosaicFactory subclass methods. 
I'm going to remove it, thanks.